ocaml/asmcomp/cmmgen.ml

```diff
@@ -1331,35 +1328,23 @@ and transl_prim_3 env p arg1 arg2 arg3 dbg =
 
   (* Effects *)
   | Presume ->
-      Misc.fatal_error "Effects-related primitives not yet supported"
-      (*
-      (* CR mshinwell: Rc_normal may be wrong, but this code is unlikely
-         to be in production by then *)
+      (* CR mshinwell: check Rc_normal *)
```

Comment:
`Rc_normal` is _required_ here, because there are some usages of effects with repeated resumes, and these should consume `O(1)` stack space by tail-calling `caml_resume`.